Cargo.toml

```diff
@@ -43,4 +42,5 @@ name = "headless_chrome"
 path = "src/lib.rs"
 
 [features]
+default = [ "reqwest", "directories", "zip" ]
```

Comment:
Could we make this a named feature in itself instead of dispersed in `default`? Something like `fetch`

Reply:
yeah of course, good idea.